steps/device-cloud-for-maestro/0.0.10/step.yml

```diff
@@ -0,0 +1,250 @@
+title: Device Cloud for Maestro
+summary: Uses devicecloud.dev to execute Maestro mobile tests
+description: Executes Maestro mobile tests on devicecloud.dev with configurable parameters
+website: https://devicecloud.dev
+source_code_url: https://github.com/devicecloud-dev/bitrise-integration
+published_at: 2025-04-08T16:43:49.443528+01:00
```

Comment:
You can add issue tracker using `support_url`, example: https://github.com/bitrise-steplib/steps-xcode-test/blob/d0bfcffbf915da8217c3286629ece67c49bcd084/step.yml#L29